packages/esm-patient-chart-app/src/workspace/workspace-window.component.tsx

```diff
@@ -73,12 +77,22 @@ const WorkspaceWindow: React.FC<ContextWorkspaceParams> = () => {
         aria-label="Workspace Title"
         className={`${styles.header} ${maximized ? `${styles.fullWidth}` : `${styles.dynamicWidth}`}`}
       >
+        {layout === 'tablet' && !canHide && (
```

Comment:
So if `!canHide` we want to render the "Hide" icon?

Reply:
Sorry, it is the close button, wrong iconDescription